src/Entity/ProductCommentCriterion.php

```diff
@@ -108,4 +121,25 @@ public function setActive($active)
 
         return $this;
     }
+
+    /**
+     * @return ArrayCollection
+     */
+    public function getLanguages(): ArrayCollection
```

Comment:
Watchout, can be also a PersistentCollection (happened for me in a side project :sweat_smile: )